backend/de.metas.edi/src/main/java/de/metas/edi/api/impl/pack/EDIDesadvPackService.java

```diff
@@ -145,30 +144,6 @@ public EDIDesadvPackService.Sequences createSequences(@NonNull final EDIDesadvId
 		return new EDIDesadvPackService.Sequences(packSeqNoSequence, packItemLineSequence);
 	}
 
-	@NonNull
-	public I_M_HU_PI_Item_Product extractHUPIItemProduct(final I_C_Order order, final I_C_OrderLine orderLine)
```

Comment:
(for next time)

IMHO we could keep this method and simply call hupiItemProductBL.extractHUPIItemProduct from it

benefit:
* we won't have to introduce a new dependency (the IHUPIItemProductBL)  in places where this method was used
* remains fairly simple from outside